uint8 PASSIVE = 0
uint8 RAW = 1
uint8 FOLLOW = 2
uint8 TWIST = 3
uint8 GYRO = 4

uint8 mode
geometry_msgs/Accel accel
float64 current_limit